content/statsServiceMap.md

```diff
@@ -7,7 +7,6 @@ title: Using the dataRetrieval Stats Service
 categories: Data Science
 tags: 
   - R
-  - dataRetrieval
```

Comment:
I modified the package to always add R....but I didn't figure out a way to add an argument that the user could define(ie...dataRetrieval or geoknife)...if someone wants to mess with the package, that would be great:

https://github.com/USGS-R/USGSmarkdowntemplates